### PR TITLE
azure storage options

### PR DESCRIPTION
**Pull request by mohanreddypmr**
while opting azure blob as input dir , we are not passing azure storage options to azure blob client , I modified those changes.

### CODE REVIEW

**Comment by deependujha** [comment]
Hey @mohanreddypmr, appreciate your willingness to make litdata better.

However, there's an ongoing PR https://github.com/Lightning-AI/litdata/pull/348 that aims to fix many similar issues.

Feel free to review it. And, looking forward to more contributions from you.

**Comment by tchaton** [comment]
> Hey @mohanreddypmr, appreciate your willingness to make litdata better.
> 
> However, there's an ongoing PR #348 that aims to fix many similar issues.
> 
> Feel free to review it. And, looking forward to more contributions from you.

@deependujha Let's land this one to unblock him until your work is merged.

**Review by tchaton — APPROVED** [review]

**Pull request merged**
